airgun/entities/hardware_model.py

```diff
@@ -48,7 +48,7 @@ def delete(self, entity_name, err_message=''):
         view.table.row(name=entity_name)['Actions'].widget.click()
         view.delete_dialog.confirm()
         if err_message:
-            view.flash.assert_message(err_message)
+            view.flash.assert_message(f"Danger alert: {err_message}")
```

Comment:
Why are hardcoding here? It should be coming from the test?

Reply:
This part of the notification is actually not visible, but the text is here to indicate the error. I think it makes more sense to have it hardcoded here than in the test.